### PR TITLE
Changelog v1.21.0

### PR DESCRIPTION
**Pull request by eladlachmi**
Update changelog for v1.21.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+# v1.21.0
+
+:new: What's new:
+
+- Refuse to create lakectl locals on case-insensitive filesystems (#7650)
```
Comment (N-o-Z) [inline CHANGELOG.md:7]:
```suggestion
- Refuse to create lakectl local on case-insensitive filesystems (#7650)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
...
+- S3 gateway - return slowdown on DynamoDB throttle (#7685)
+- UI: calculate total size of common prefix (#7720)
+- lakectl added support for retries (#7723)
```
Comment (N-o-Z) [inline CHANGELOG.md:10]: maybe batch with the previous?

**Review by N-o-Z — APPROVED** [review]
Approved with comments

File: CHANGELOG.md (hunk 1 above)
Comment (eladlachmi) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 2 above)
Comment (eladlachmi) [reply CHANGELOG.md]: Done

**Pull request merged**